jest.config.js

```diff
@@ -19,7 +19,7 @@ const config = {
   // Those packages are ESM, so we need them to be processed by Babel
   transformIgnorePatterns: ['/node_modules/(?!(redent|strip-indent)/)'],
   coverageDirectory: '<rootDir>/coverage',
-  moduleDirectories: ['<rootDir>/node_modules', '<rootDir>/app/javascript'],
+  moduleDirectories: ['node_modules', '<rootDir>/app/javascript'],
```

Comment:
Sounds good, but shouldn't it be `app/javascript` as well?

Reply:
If I understand the core issue correctly, having an absolute `node_modules` reference here forces Jest to resolve the modules to `<rootDir>/node_modules/<module>`, which is incorrect when some non-direct dependencies require a specific version, which can be in `node_modules/direct-dep/node_modules/indirect-dep`. Specifying it as relative allows Node to correctly resolve the correct version, because it uses the `node_modules/` directory relative to the initial dependency.

For `app/javascript`,  we do not need this behaviour, and we know the exact location, so it is best to use it.